src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Commands/SubmitCommitmentTests/WhenSubmittingCommitment.cs

```diff
@@ -159,7 +163,133 @@ public void Setup()
+        }
+
+        [Test]
+        public async Task ShouldCallSendNotificationCommandForTransferCohortFirstApproval()
+        {
+            const string legalEntityName = "Receiving Employer Ltd";
+
+            const string template =
+                @"Cohort ((cohort_reference)) is ready to review. ((receiving_employer)) has chosen to use funds transferred from another employer to pay for the training in this cohort.
+
+What you need to know about cohorts funded through transfers
+
+You and the employer will approve the cohort as usual. The cohort will then be sent to the employer who is transferring the funds for final approval. Once the cohort has been approved you will be able to view and manage the apprentices
+To review cohort ((cohort_reference)) you will need to sign in to your apprenticeship service account at https://providers.apprenticeships.sfa.bis.gov.uk.
+
+This is an automated message. Please don’t reply to this email.
+
+Kind regards,
+
+Apprenticeship service team";
+
+            string expectedEmailBody = $@"Cohort {CohortReference} is ready to review. {legalEntityName} has chosen to use funds transferred from another employer to pay for the training in this cohort.
+
+What you need to know about cohorts funded through transfers
+
+You and the employer will approve the cohort as usual. The cohort will then be sent to the employer who is transferring the funds for final approval. Once the cohort has been approved you will be able to view and manage the apprentices
+To review cohort {CohortReference} you will need to sign in to your apprenticeship service account at https://providers.apprenticeships.sfa.bis.gov.uk.
+
+This is an automated message. Please don’t reply to this email.
+
+Kind regards,
+
+Apprenticeship service team";
+
+            SendNotificationCommand arg = null;
+            _validCommand.LastAction = LastAction.Approve;
+            _repositoryCommitment.TransferSender = new TransferSender();
+            _repositoryCommitment.LegalEntityName = legalEntityName;
+
+            _mockEmailLookup
+                .Setup(x => x.GetEmailsAsync(It.IsAny<long>(), It.IsAny<string>()))
+                .ReturnsAsync(new List<string> { "test@email.com" });
+
+            _mockMediator.Setup(x => x.SendAsync(It.IsAny<SendNotificationCommand>()))
+                .ReturnsAsync(new Unit()).Callback<SendNotificationCommand>(x => arg = x);
+
+            await _handler.Handle(_validCommand);
+
+            _mockMediator.Verify(x => x.SendAsync(It.IsAny<SendNotificationCommand>()));
+            arg.Email.TemplateId.Should().Be("TransferProviderCommitmentNotification");
+            arg.Email.Tokens["cohort_reference"].Should().Be(CohortReference);
+            arg.Email.Tokens["receiving_employer"].Should().Be(legalEntityName);
+
+            var emailBody = PopulateTemplate(template, arg.Email.Tokens);
+            TestContext.WriteLine(emailBody);
+            Assert.AreEqual(expectedEmailBody, emailBody);
```

Comment:
Is this something we're interested in testing? We don't handle the actual templating.

Reply:
I was in two minds about whether to test the template, but I added it because
a) I wanted to check that the template didn't contain any errors, such as misspelled substitution tokens
b) the templates are stored in a system we don't own, so having the templates in the tests allows us to keep a record of the template in source control (if we ever need to recover it)